views/github-create-branch.hbs

```diff
@@ -63,7 +63,8 @@
     <div class="field-group">
       <aui-label for="branch-name">Branch name</aui-label>
       <input class="text full-width-field" type="text" data-aui-validation-field required name="branch-name"
-             placeholder="Enter branch name"/>
+        placeholder="Enter branch name" data-aui-validation-pattern="\w*{{issue.key}}\w*"
```

Comment:
Can you add a message for `This field is required.` tooooo
<img width="899" alt="Screen Shot 2022-09-07 at 10 55 41 AM" src="https://user-images.githubusercontent.com/3848691/188765333-90bd12c7-00f9-4751-9112-20369c7da163.png">

Reply:
Message added !!